### PR TITLE
refactor: support only docker for multi arch image build

### PR DESCRIPTION
**Pull request by kmehant**
Signed-off-by: Mehant Kammakomati <kmehant@gmail.com>

### CODE REVIEW

**Comment by kmehant** [comment]
@ashokponkumar refactored it here, please review!

File: Makefile [hunk 1]
```diff
@@ -199,11 +199,9 @@ endif
 
 .PHONY: cmultibuildpush
 cmultibuildpush: ## Build and push multi arch container image
-ifndef CONTAINER_TOOL
-	$(error No container tool (docker, podman) found in your environment. Please, install one)
+ifndef DOCKER_CMD
+	$(error Presently, only docker is supported for multi architecture image build)
```
Comment (ashokponkumar) [inline Makefile:203]: Also mention that docker wasn't detected. Also add a comment to instruct them to install docker and try again

**Review by ashokponkumar — CHANGES_REQUESTED** [review]

File: Makefile (hunk 1 above)
Comment (ashokponkumar) [reply Makefile:203]: "Docker wasn't detected. Please install docker and try again."

**Comment by kmehant** [comment]
@ashokponkumar @HarikrishnanBalagopal please review!

**Review by ashokponkumar — APPROVED** [review]

**Pull request merged**